### PR TITLE
Update .travis.yml. Fixes #26

### LINKED ISSUE #26: Add latest node to travis.yml
Not sure how Travis works but it looks like only the latest iojs is tested in your repo.
From what I understand, [`node_js` environments are passed to `nvm`](https://docs.travis-ci.com/user/ci-environment/#Node.js-VM-images)
So you may want to add [`node`](https://github.com/creationix/nvm#usage) (why not remove `iojs`), because I'm not sure anyone cares any longer.

### PR DESCRIPTION
**Pull request by DavidBruant**
[No idea what I'm doing](https://s-media-cache-ak0.pinimg.com/736x/7c/40/d9/7c40d91b31d24cc6836ac80297caa7b6.jpg

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - "0.10"
+  - '0.10'
   - '0.12'
-  - 'iojs'
+  - 'node'
```
Comment (mafintosh) [inline .travis.yml]: replace node with `'4'` and add `'5'` on another line

Comment (DavidBruant) [inline .travis.yml]: Removing `node` would mean that any time there is a new node version, you'll have to go back to the repo to update the .travis.yml.
What about : 
````
   - '0.10'
   - '0.12'
   - '4'
   - '5'
   - 'node'
````
?

Comment (mafintosh) [inline .travis.yml]: Sure!

Comment (DavidBruant) [inline .travis.yml]: Sounds good!
I need to go. I'll force-push to this branch later tonight or tomorrow.

**Comment by DavidBruant** [comment]
I force-pushed following the comments.

**Pull request merged**